paper.md

```diff
@@ -208,74 +193,55 @@ competency of students on the lesson material.
 
 We have found that this combination of clear lesson outlines, coding submodules,
 interleaved exercises, and summative assignments gave each lecture a
-predictable structure. It also assures that lesson objectives
+predictable structure. It also ensured that lesson objectives
 were both covered and conveyed effectively in order for students to
 complete both the exercises and assignments.
 
 Each of our lectures built on skills and concepts that would ultimately allow
 students to work on a final open-ended analysis of real open ecological data.
-Of note, the datasets we chose for the course were large and messy (e.g. missing
-values), reflecting the type of data that is being increasingly generated across
+We deliberately chose large and messy (e.g. missing values) datasets for use in the course, 
+reflecting the types of data that are being increasingly generated across
 various disciplines. With this goal in mind, we designed lectures to provide the
 building blocks to clean, manipulate, visualize and analyze any datasets
 students may come across, including those used for the final projects.
 
-[^1]: *A note on "live-coding".* Live-coding is a hands-on method of teaching
-coding where the instructor shares their screen with the class and types all
-commands on their computer while the group follows along. Live-coding has proven
-to be a very effective teaching technique as it forces the instructor to go
-slowly and ensures that participants get to try out every command being used.
-Furthermore, mistakes and errors are bound to happen. We used two colors of
-sticky notes, distributed to students each class, one color to signal
-whether they are ready to move on, the other to signal that they are
-stuck/have a question, whereby another instructor in the class can follow
-up with the student as the class continues. It allows learners to
-experience common errors themselves and debug them in a supportive
-environment, to explore variations on material as they go, and to
-immediately check their understanding by trying things hands-on.
-See [Teaching Experience](#teaching-experience) for more details.
-This teaching format is heavily influenced by [Software Carpentry](https://software-carpentry.org/blog/2016/04/tips-tricks-live-coding.html)
-which uses live-coding as a key practice in their introductory programming
-workshops.
-
```

Comment:
Not sure the whole note [1] should be deleted but I agree the text could be trimmed down. What about removing this, as suggested here, and adding a brief sentence where this note is referenced in-text describing the process of live coding. Something like: "...taught in class using live coding, a hands-on method of teaching whereby instructors share their screens and students follow along. This approach is frequently used by organizations that teach programming (e.g. software carpentry, [link][ref])